docs/source/convex-hull.rst

```diff
@@ -0,0 +1,46 @@
+Convex Hull
+==========
+
+Used to find the smallest convex polygon that envolves a set of points in a 2D plane.
+
+.. note::
+
+    Tested on: URI1982
+
+.. code-block:: cpp
+
+    
+	struct Point{
```

Comment:
Should be easier to use a std::pair ? I know that the syntax is worse, but the amount of code would be reduced